cognite/client/data_classes/transformations/__init__.py

```diff
@@ -215,21 +215,21 @@ def try_get_or_create_nonce(oidc_credentials: OidcCredentials | None, project: s
                 else:
                     credentials = None
                 try:
-                    session = self._cognite_client.iam.sessions.create(credentials, project=project)
+                    session = self._cognite_client.iam.sessions.create(credentials)
```

Comment:
I think this is what's missing
```suggestionclient = self._cognite_client
                    if project != self._cognite_client.config.project:
                        cnf = copy(self._cognite_client.config)
                        cnf.project = project
                        client = CogniteClient(cnf)
                    session = client.iam.sessions.create(credentials)
```

Reply:
created #1352 just to confirm it works